src/ServerlessWorkflow.Sdk/ActionExecutionMode.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * Copyright 2020-Present The Serverless Workflow Specification Authors
```

Comment:
```suggestion
 * Copyright 2021-Present The Serverless Workflow Specification Authors
```

Sorry Charles, could you please change the headers?

Reply:
Yes, I will fix that tomorrow morning, no problem ;)

Reply:
I'm not sure what has changed in 0.6 from latest 0.5x, but I will make sure to update whatever needs to be.

Reply:
@ricardozanini I just reviewed the 0.6 and it seems my code is up-to-date. I'll change the version to 0.6